modules/ROOT/pages/visual-tour/index.adoc

```diff
@@ -48,9 +48,18 @@ See xref:query/visual-tour.adoc#copilot[Query co-pilot] and xref:explore/explore
 
 image::enable-copilot.png[]
 
+==== Settings for GDS sessions
+
+include::partial$EAP.adoc[]
+
+The following configuration settings are available for link:{gds-sessions-page}[GDS Sessions]:
+
+* The maximum available memory for each Session
+* The maximum number of concurrent Sessions within the organization
+
```

Comment:
(not this PR) also saw we could mention single-sign-on configuration here on the org settings

![image](https://github.com/user-attachments/assets/0cfe2191-8b0c-47f0-aa75-a8328eb3a31a)

Reply:
Sure, I'll leave this to @AlexicaWright.

Reply:
Kindly flesh this out a little for me.